app/components/article-featured-video.js

```diff
@@ -25,6 +36,20 @@ export default Component.extend({
 		this.initVideoPlayer();
 	})),
 
+	actions: {
+		/**
+		 * FIXME FEATURED VIDEO A/B TEST ONLY
+		 */
+		dismissPlayer() {
+			this.element.classList.remove(scrollClassName);
+
+			this.player.setMute(true);
```

Comment:
why is that? I think we should rather pause the video instead of muting it.

Reply:
```
Stanley Tran [10:24 PM] 
hey there - let’s mute the video when the video is dismissed. i know it’s still being tricky, but i think it’s a step better than what they’re doing on BBC’s mobile site which still has it playing unmuted. dzieki!
```